concepticondata/conceptlists/Zhang-2019-308.tsv

```diff
@@ -0,0 +1,309 @@
+Zhang-2019-308-86	86	the joint	joint	999	JOINT	
+Zhang-2019-308-87	87	be late	late	425	BE LATE	
+Zhang-2019-308-88	88	the louse	louse	1392	LOUSE	
+Zhang-2019-308-89	89	the milk, nipple	milk, nipple	2129	BREAST OR MILK	
+Zhang-2019-308-90	90	the millet	millet	931	MILLET	
+Zhang-2019-308-91	91	the millet or rice grains	millet or rice grains	931	MILLET	
+Zhang-2019-308-92	92	the mother's brother	mother's brother	2692	MATERNAL UNCLE (MOTHER'S BROTHER)	
+Zhang-2019-308-93	93	the mushroom	mushroom	641	MUSHROOM	
+Zhang-2019-308-94	94	the nail	nail	502	NAIL (TOOL)	
+Zhang-2019-308-95	95	the name	name	1405	NAME	
+Zhang-2019-308-96	96	be near	near	1942	NEAR	
+Zhang-2019-308-97	97	the needle	needle	1382	NEEDLE (FOR SEWING)	
+Zhang-2019-308-98	98	the night	night	1233	NIGHT	
+Zhang-2019-308-99	99	nine	nine	1483	NINE	
+Zhang-2019-308-100	100	not to be	not be	1240	NOT	
+Zhang-2019-308-101	101	one	one	1493	ONE	
+Zhang-2019-308-102	102	the onion	onion	2366	ONION	
+Zhang-2019-308-103	103	the owl	owl	735	OWL	
+Zhang-2019-308-104	104	the ox	ox	1169	OX	
+Zhang-2019-308-105	105	to peel off	peel off	275	PEEL	
+Zhang-2019-308-106	106	the pig	pig	1337	PIG	
+Zhang-2019-308-107	107	the pillow	pillow	1044	PILLOW	
+Zhang-2019-308-108	108	the plant root	plant root	670	ROOT	
+Zhang-2019-308-109	109	the pliers	pliers			
+Zhang-2019-308-110	110	the poison	poison	655	POISON	
+Zhang-2019-308-111	111	the pulse, beans	pulse, beans	832	BEAN	
+Zhang-2019-308-112	112	the rabbit, hare	rabbit, hare	2345	LEPORID (RABBIT OR HARE)	
+Zhang-2019-308-113	113	be rich	rich	712	RICH	
+Zhang-2019-308-114	114	the rope	rope	1218	ROPE	
+Zhang-2019-308-115	115	the segment	segment	594	PIECE	
+Zhang-2019-308-116	116	seven	seven	1704	SEVEN	
+Zhang-2019-308-117	117	the shame	shame	1776	SHAME	
+Zhang-2019-308-118	118	the sheep	sheep	1331	SHEEP	
+Zhang-2019-308-119	119	the shell	shell	598	SHELL	
+Zhang-2019-308-120	120	the shoulder	shoulder	1482	SHOULDER	
+Zhang-2019-308-121	121	the shoulder blade	shoulder blade	172	SHOULDERBLADE	
+Zhang-2019-308-122	122	be single	single	1493	ONE	"It is actually ""one"" in Sino-Tibetan."
+Zhang-2019-308-123	123	six	six	1703	SIX	
+Zhang-2019-308-124	124	the snake brood	snake brood			
+Zhang-2019-308-125	125	the son-in-law	son-in-law	1056	SON-IN-LAW	
+Zhang-2019-308-126	126	be sour	sour	1906	SOUR	
+Zhang-2019-308-127	127	the sovereign, queen	sovereign, queen	1306	QUEEN	
+Zhang-2019-308-128	128	be stiff	stiff			
+Zhang-2019-308-129	129	to strike	strike	353	STRIKE	
+Zhang-2019-308-130	130	the string, cord	string, cord	362	STRING	
+Zhang-2019-308-131	131	be strong, hard	strong, hard	1884	HARD	
+Zhang-2019-308-132	132	to suffer defeat	suffer defeat	1660	LOSE	
+Zhang-2019-308-133	133	the tail	tail	1220	TAIL	
+Zhang-2019-308-134	134	there is no X such that X...	there is no X such that X...			don't know what to do with this one
+Zhang-2019-308-135	135	the thighbone	thighbone	800	THIGH	
+Zhang-2019-308-136	136	be thin	thin	2249	THIN (OF SHAPE OF OBJECT)	
```

Comment:
Is the specification (OF SHAPE OF OBJECT) justified by the meaning of the target lexemes? If yes, then ignore my comment.